### PR TITLE
Story portfolio v1 - Review

### PR DESCRIPTION
**Pull request by jdselby24**

### CODE REVIEW

**Comment by charliecog** [comment]
commit messages are good - couple of them could do with a little more detail about what you have actually done eg: "update"

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,409 @@
+/*        O__O
+          (oo)
+   /-------\/
+  / |     ||
+ *  ||----||
+    ~~    ~~
+Cow at Disneyland
```
Comment (charliecog) [inline styles.css]: remooooooove

File: index.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+<head>
+	<meta name="viewport" content="width=device-width">
+	
+	<title>Josh Selby</title>
+
+	<script src="https://kit.fontawesome.com/dd9a23d139.js"></script>
+	<link rel="stylesheet" href="https://use.typekit.net/gxz6jar.css"> <!-- link to font 'din-condensed' -->
+	<link href="https://fonts.googleapis.com/css?family=Lora:400,700&display=swap" rel="stylesheet"> <!-- link to font 'lora' -->
+	<link rel="stylesheet" type="text/css" href="normalize.css">
```
Comment (charliecog) [inline index.html]: the links are all self closing tags - missing / at the end

File: index.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+
+	<nav>
+		<div id="logo">J S</div>
```
Comment (charliecog) [inline index.html]: why id? - could be class

File: index.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+		<ul>
+			<li><a href="#contact-section">Contact Me</a></li>
+			<li><a href="#portfolio-item-box">Portfolio</a></li>
+			<li><a href="#about-text">About Me</a></li>
+			<li>
+				<div id="hamburger">
```
Comment (charliecog) [inline index.html]: class not id

File: index.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+					<div class="hamburgerRow" id="hRow1"></div>
+					<div class="hamburgerRow" id="hRow2"></div>
+					<div class="hamburgerRow" id="hRow3"></div>
+				</div>
+			</li>
+		</ul>
+	</nav>
+
+	<section class="hero">
```
Comment (charliecog) [inline index.html]: header instead of section

File: index.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+	<nav>
+		<div id="logo">J S</div>
+
+		<ul>
+			<li><a href="#contact-section">Contact Me</a></li>
+			<li><a href="#portfolio-item-box">Portfolio</a></li>
+			<li><a href="#about-text">About Me</a></li>
+			<li>
+				<div id="hamburger">
+					<div class="hamburgerRow" id="hRow1"></div>
```
Comment (charliecog) [inline index.html]: why id's?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+	<section class="hero">
+		<section class="container">
+			<article id="hero-text">
+				<h1>Josh Selby</h1>
+				<h3>Full stack web developer</h3>
+				<h6>guitar player, photographer, car mechanic..</h6>
+			</article>
+
+			<a id="hero-arrow" href="#about-text"><div>\/</div></a>
```
Comment (charliecog) [inline index.html]: ids

File: index.html [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+		</section>
+	</section>
+
+	<section class="about">
+		<section class="container">
```
Comment (charliecog) [inline index.html]: too many levels of section, just put about and container on the same section

File: index.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+			<article id="about-text">
```
Comment (charliecog) [inline index.html]: also you dont need the article can just text-align center the parent 'about' section

File: index.html [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+				<h1>About Me</h1>
+				<p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+			</article>
+		</section>
+	</section>
+
+	<section class="portfolio">
+		<section class="container">
+			<article class="portfolio-text">
```
Comment (charliecog) [inline index.html]: delete article and put the class on the parent

File: index.html [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+	<section class="about">
+		<section class="container">
+			<article id="about-text">
+				<h1>About Me</h1>
+				<p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+			</article>
+		</section>
+	</section>
+
+	<section class="portfolio">
```
Comment (charliecog) [inline index.html]: remove the section and put class on the container

File: index.html [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+						<div class="coming-soon" id="pf-item-6-img"></div>
+						<h3>Portfolio Item 6</h3>
+						<p><a href="#portfolio-item-box">View</a> | <a href="#portfolio-item-box">Github</a></p>
+					</article>
+				</div>
+			</article>
+		</section>
+	</section>
+
+	<!-- \/ for future use by contact \/ -->
```
Comment (charliecog) [inline index.html]: remove

File: index.html [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+
+	<section class="contact">
+		<section class="container">
```
Comment (charliecog) [inline index.html]: could be combined into one section

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,409 @@
...
+i {
+	color: #000000;
+	font-size: 40px;
+}
+
+/*          (__)
+            (@@)
+     /-------\/
+    / |     ||
+   *  ||----||
```
Comment (charliecog) [inline styles.css]: remove

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,409 @@
...
+}
+
+nav ul li a:hover {
+	border: .5px solid #000000;
+	border-radius: 5px;
+	padding: 6.5px;
+}
+
+/*             )\               (__)
+              /  \              (oo)
```
Comment (charliecog) [inline styles.css]: remove

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,409 @@
...
+           Cow swimming at Amity Island
+(Where Jaws was set, for those less educated)*/
+
+/* HERO STYLES */
+
+.hero {
+	margin: 0;
+	color: #ffffff;
+	background-color: #000000;
+	background-image: url(images/hero.jpg);
```
Comment (charliecog) [inline styles.css]: need "" around the url

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,409 @@
...
+}
+
+#hero-arrow:hover {
+	color: #5f5f5f;
+}
+
+/*          (__)
+            ($$)
+     /-------\/
+    / |=====||
```
Comment (charliecog) [inline styles.css]: oh my god remove

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,409 @@
...
+
+#about-text p {
+	font-size: 28px;
+	font-weight: 200;
+}
+
+/*               \_|_/
+                  (oo)
+           /-------\/
+          / |     ||
```
Comment (charliecog) [inline styles.css]: please stop

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,409 @@
...
+	border-radius: 10px;
+}
+
+.portfolio-item a {
+	font-size: 20px;
+}
+
+/*     \^^^^^^^^\   (__)
+        \^^^^^^^^\\ (oo)
+     *-----\_______\/\/
```
Comment (charliecog) [inline styles.css]: remove

File: styles.css [hunk 20]
```diff
@@ -0,0 +1,409 @@
...
+
+.portfolio-item div {
+	background-position: center;
+  	background-repeat: no-repeat;
+  	background-size: cover;
+  	position: relative;
+}
+
+.coming-soon {
+	background-image: url(images/coming-soon.png);
```
Comment (charliecog) [inline styles.css]: ""

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,409 @@
...
+}
+
+#pf-item-1-img {
+	background-image: url(images/item1.png);
```
Comment (charliecog) [inline styles.css]: ""

File: styles.css [hunk 22]
```diff
@@ -0,0 +1,409 @@
...
+}
+
+
+/*                   (__)
+                     (oo)
+  /-------------------\/
+ / |                 ||
```
Comment (charliecog) [inline styles.css]: nope

File: styles.css [hunk 23]
```diff
@@ -0,0 +1,409 @@
...
+
+
+.contact .container {
+	text-align: center;
+}
+
+/*     \ (__)
+       \\(oo)
+  /-----\\\/
+ / |    (##)
```
Comment (charliecog) [inline styles.css]: remove

File: styles.css [hunk 24]
```diff
@@ -0,0 +1,409 @@
...
+
+@media screen and (max-width: 1400px) {
+	.portfolio .container {
+		width: 95%;
+	}
+}
+
+@media screen and (max-width: 1200px) {
+	.contact article {
+		/*display: block;*/
```
Comment (charliecog) [inline styles.css]: remove comment

File: styles.css [hunk 25]
```diff
@@ -0,0 +1,409 @@
...
+
+	.portfolio-item div {
+		width: 320px;
+		height: 170px;
+	}
+
+}
+
+/*    vv    vv
+      ||----||  *
```
Comment (charliecog) [inline styles.css]: remove

File: styles.css [hunk 26]
```diff
@@ -0,0 +1,409 @@
...
+    ~~    ~~
+Cow at Disneyland
+*/
+
+
+/* MASTER STYLES */
+* {
+	font-family: din-condensed, sans-serif;
+	font-weight: 400;
+	font-style: normal;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 27]
```diff
@@ -0,0 +1,409 @@
...
+  / |     ||
+ *  ||----||
+    ~~    ~~
+Cow at Disneyland
+*/
+
+
+/* MASTER STYLES */
+* {
+	font-family: din-condensed, sans-serif;
```
Comment (charliecog) [inline styles.css]: needs ''

File: styles.css [hunk 28]
```diff
@@ -0,0 +1,409 @@
...
+   *  ||----||
+      ^^    ^^
+   Cow who ate
+psychadelic mushrooms*/
+
+/*NAVBAR STYLES*/
+
+nav {
+	overflow: auto;
+	background-color: #ffffff;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 29]
```diff
@@ -0,0 +1,409 @@
...
+	color: #000000;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 30]
```diff
@@ -0,0 +1,409 @@
...
+	font-size: 25px;
+	height: 70px;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 31]
```diff
@@ -0,0 +1,409 @@
...
+	}
+
+nav #logo {
+	float: left;
+	font-size: 40px;
+	padding: 8px;
+	font-weight: 600;
+	margin-left: 30px;
```
Comment (charliecog) [inline styles.css]: margin can be declared all in one line

File: styles.css [hunk 32]
```diff
@@ -0,0 +1,409 @@
...
+/*             )\               (__)
+              /  \              (oo)
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+           Cow swimming at Amity Island
+(Where Jaws was set, for those less educated)*/
+
+/* HERO STYLES */
+
+.hero {
+	margin: 0;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 33]
```diff
@@ -0,0 +1,409 @@
...
+	color: #ffffff;
+	background-color: #000000;
+	background-image: url(images/hero.jpg);
+	background-position: center;
+  	background-repeat: no-repeat;
+  	background-size: cover;
+  	position: relative;
```
Comment (charliecog) [inline styles.css]: dont think its doing anything

File: styles.css [hunk 34]
```diff
@@ -0,0 +1,409 @@
...
+	height: 870px;
+}
+
+#hero-text {
+	margin: 0 0 0 20px;
+	font-size: 46px;
+	font-family: 'Lora', serif;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 35]
```diff
@@ -0,0 +1,409 @@
...
+
+.hero h6 {
+	margin: 0;
+	font-size: 40%;
+}
+
+#hero-arrow {
+	display: inline-block;
+	position: relative;
+	text-decoration: none;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 36]
```diff
@@ -0,0 +1,409 @@
...
+	color: #ffffff;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 37]
```diff
@@ -0,0 +1,409 @@
...
+	font-size: 50px;
+	font-weight: 700;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 38]
```diff
@@ -0,0 +1,409 @@
...
+
+#about-text {
+	text-align: center;
+	padding: 52px;
+}
+
+#about-text h1 {
+	font-size: 44px;
+	margin: 0;
+	padding-top: 20px;
```
Comment (charliecog) [inline styles.css]: just add this padding to the parents padding rather than have it over two places

File: styles.css [hunk 39]
```diff
@@ -0,0 +1,409 @@
...
+	font-size: 54px;
+}
+
+.portfolio h3 {
+	font-size: 25px;
+	margin-bottom: 5px;
+}
+
+.portfolio-text {
+	padding-top-top: 20px;
```
Comment (charliecog) [inline styles.css]: this isn't valid css

File: styles.css [hunk 40]
```diff
@@ -0,0 +1,409 @@
...
+Stretch Cow*/
+
+/* CONTACT STYLES */
+.contact h2 {
+	text-align: center;
+}
+
+.contact article {
+	display: inline-block;
+	text-align: center;
```
Comment (charliecog) [inline styles.css:201]: not doing anything

File: styles.css [hunk 41]
```diff
@@ -0,0 +1,409 @@
...
+	width: auto;
```
Comment (charliecog) [inline styles.css]: not doing anything

**Review by charliecog — COMMENTED** [review]
General comments:
Contact button doesn't link to the section
Use float in future
Use rems in future

Looks good - almost there

File: index.html (hunk 2 above)
Comment (jdselby24) [reply index.html]: added

File: index.html (hunk 3 above)
Comment (jdselby24) [reply index.html]: changed

File: index.html (hunk 4 above)
Comment (jdselby24) [reply index.html]: changed

File: index.html (hunk 6 above)
Comment (jdselby24) [reply index.html]: removed

File: index.html (hunk 5 above)
Comment (jdselby24) [reply index.html]: changed

File: index.html (hunk 7 above)
Comment (jdselby24) [reply index.html]: changed

File: index.html (hunk 8 above)
Comment (jdselby24) [reply index.html]: fixed

File: index.html (hunk 11 above)
Comment (jdselby24) [reply index.html]: needed for black background to span the whole page

File: index.html (hunk 10 above)
Comment (jdselby24) [reply index.html]: done

File: index.html (hunk 12 above)
Comment (jdselby24) [reply index.html]: done

File: index.html (hunk 13 above)
Comment (jdselby24) [reply index.html]: done

File: styles.css (hunk 1 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 14 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 15 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 17 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 18 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 39 above)
Comment (jdselby24) [reply styles.css]: fixed

File: styles.css (hunk 19 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 22 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 23 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 25 above)
Comment (jdselby24) [reply styles.css]: remoooooooved

File: styles.css (hunk 27 above)
Comment (jdselby24) [reply styles.css]: fixed

File: styles.css (hunk 26 above)
Comment (jdselby24) [reply styles.css]: fixed

File: styles.css (hunk 28 above)
Comment (jdselby24) [reply styles.css]: removed

File: styles.css (hunk 29 above)
Comment (jdselby24) [reply styles.css]: removed

File: styles.css (hunk 30 above)
Comment (jdselby24) [reply styles.css]: removed

File: styles.css (hunk 31 above)
Comment (jdselby24) [reply styles.css]: done

File: styles.css (hunk 32 above)
Comment (jdselby24) [reply styles.css]: removed

File: styles.css (hunk 16 above)
Comment (jdselby24) [reply styles.css]: added

File: styles.css (hunk 33 above)
Comment (jdselby24) [reply styles.css]: removed

File: styles.css (hunk 35 above)
Comment (jdselby24) [reply styles.css]: hero arrow removed anyway due to dubious position

File: styles.css (hunk 36 above)
Comment (jdselby24) [reply styles.css]: hero arrow removed anyway due to dubious position

File: styles.css (hunk 37 above)
Comment (jdselby24) [reply styles.css]: hero arrow removed anyway due to dubious position

File: styles.css (hunk 38 above)
Comment (jdselby24) [reply styles.css]: done

File: styles.css (hunk 20 above)
Comment (jdselby24) [reply styles.css]: done

File: styles.css (hunk 21 above)
Comment (jdselby24) [reply styles.css]: done

File: styles.css (hunk 40 above)
Comment (jdselby24) [reply styles.css:201]: yes it does, aligns text only visible on hover

File: styles.css (hunk 41 above)
Comment (jdselby24) [reply styles.css]: removed

File: styles.css (hunk 24 above)
Comment (jdselby24) [reply styles.css]: gone

**Review by charliecog — APPROVED** [review]
woop woop

**Pull request merged**